docs/sources/flow/reference/components/loki.source.syslog.md

```diff
@@ -0,0 +1,144 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/loki.source.syslog
+title: loki.source.syslog
+---
+
+# loki.source.syslog
+
+`loki.source.syslog` listens for syslog messages over TCP or UDP connections
+and forwards them to other `loki.*` components. The messages must be compliant
+with the [RFC5424](https://www.rfc-editor.org/rfc/rfc5424) format.
+
+The component starts a new syslog listener for each of the given `config`
+blocks and fans out incoming entries to the list of receivers in `forward_to`.
+
+Multiple `loki.source.syslog` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+loki.source.syslog "LABEL" {
+  config {
+    listen_address = "LISTEN_ADDRESS"
+  }
+  ...
+
+  forward_to = RECEIVER_LIST
+}
+```
+
+## Arguments
+
+`loki.source.syslog` supports the following arguments:
+
+Name         | Type                   | Description          | Default | Required
+------------ | ---------------------- | -------------------- | ------- | --------
+`forward_to` | `list(LogsReceiver)`   | List of receivers to send log entries to. | | yes
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`loki.source.syslog`:
+
+Hierarchy | Name | Description | Required
+--------- | ---- | ----------- | --------
+listener | [config][] | Configures a listener for IETF Syslog (RFC5424) messages. | no
+listener > tls_config | [tls_config][] | Configures TLS settings for connecting to the endpoint for TCP connections. | no
+
+The `>` symbol indicates deeper levels of nesting. For example, `config > tls_config` 
+refers to a `tls_config` block defined inside a `config` block.
+
+[listener]: #listener-block
+[tls_config]: #tls_config-block
+
+### listener block
+
+The `listener` block defines the listen address and protocol where the listener
+will expect syslog messages to, as well as its behavior when receiving
+messages.
```

Comment:
```suggestion
The `listener` block defines the listen address and protocol where the listener
expects syslog messages to be sent to, as well as its behavior when receiving
messages.
```

Reply:
Something seemed to be missing from this sentence, so I took a guess...

Reply:
Thanks! And 🤦 about the future tense _once more_, I still haven't gotten used to it.

Reply:
> Thanks! And 🤦 about the future tense _once more_, I still haven't gotten used to it.

Haha, no worries!